classes/Domain/Services/Validator.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+use Symfony\Component\HttpFoundation\Request;
+
+interface Validator
+{
+    public function isValid(Request $request): bool;
```

Comment:
Not sure if this makes sense, are we going to implement this interface with that exact type declaration multiple times? Maybe rename it to `RequestValidator`, then?

What do you think?